dashboard/modules/logical_view/test_logical_view_head.py

```diff
@@ -136,4 +135,5 @@ def kill_actor_using_dashboard(actor):
 
 
 if __name__ == "__main__":
+    from ray.new_dashboard.tests.conftest import *  # noqa
```

Comment:
Had to move these into the main block because otherwise when packaged into a wheel, the dashboard fails to import files from test modules that aren't there. Ultimately these files should stop being imported from the dashboard at all but that will be a bigger change.